initializr-service/src/main/resources/application.yml

```diff
@@ -91,6 +96,96 @@ initializr:
     kotlin:
       version: 1.1.3
   dependencies:
+    - name: Azure
+      content:
+        - name: Azure Storage
+          id: azure-storage
+          groupId: com.microsoft.azure
+          artifactId: azure-storage-spring-boot-starter
+          description: Spring Boot starter for Azure Storage service
+          versionRange: 1.5.4.RELEASE
+          bom: azure
+          links:
+            - rel: reference
```

Comment:
Please add, if possible, two links (this apply for each entry):

* One link to a section of a developer guide that describes how to use the feature (here what is necessary to get started, what the api can do etc)
* One link to a sample/getting started guide